src/super_gradients/training/transforms/keypoints/keypoints_mixup.py

```diff
@@ -0,0 +1,52 @@
+import random
+import numpy as np
+
+from typing import Optional
+
+from super_gradients.common.registry import register_transform
+from super_gradients.training.samples import PoseEstimationSample
+
+from .abstract_keypoints_transform import AbstractKeypointTransform
+
+
+@register_transform()
+class KeypointsMixup(AbstractKeypointTransform):
+    """
+    Mix two samples together.
+
+    :attr prob:            Probability to apply the transform.
+    """
+
+    def __init__(self, prob: float):
+        """
+
+        :param prob:            Probability to apply the transform.
+        """
+        super().__init__(additional_samples_count=1)
+        self.prob = prob
+
+    def apply_to_sample(self, sample: PoseEstimationSample) -> PoseEstimationSample:
+        if random.random() < self.prob:
+            sample = self.apply_mixup(sample, sample.additional_samples[0])
+        return sample
+
+    def apply_mixup(self, sample: PoseEstimationSample, other: PoseEstimationSample) -> PoseEstimationSample:
+        sample.image = (sample.image * 0.5 + other.image * 0.5).astype(sample.image)
+        sample.mask = np.logical_or(sample.mask, other.mask).astype(sample.mask.dtype)
+        sample.joints = np.concatenate([sample.joints, other.joints], axis=0)
```

Comment:
If just one of them is None, is the padding really necessary ?
same for L40-L41